### PR TITLE
Few CTF changes.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
- Fixed that if you chose to press a cross while toggling instagib mode it will react as `Yes` instead of just canceling
- Added description for assault class (with shotgun and rocket gloves) so you know that you have faster moving speed and lover armor.
- Fixed that when you already chose a team and try to click on other teams spawners you'll be asked to consider other teams to make them equal.
- Decreased time of recharge start delay from 20 seconds to 12, because when it takes soo much to regen your shield its just faster to die and respawn with a new one.
- Added 1.7 scope to marksmans rifle.
- Little cleanup.

**Everything is arguable.**
**Ask if something is unclear or confusing.**
## Why It's Good For The Game
Less buggs, less wondering why do you die in 1 hit and others don't, a little bit of consistency.
## Changelog
:cl:
qol: CTF assault class' (with shotgun and rocket gloves)  description now mentions that it has low armor and increased moving speed.
balance: Decreased CTF recharge start delay for shields from 20 seconds to 12 seconds.
balance: Marksmans rifle now has 1.7 scope.
fix: When toggling instagib mode for CTF and pressing a cross now will just cancel instead of proceeding.
fix: When clicking on other CTF teams spawners and after you already chose your team no more will ask you to join some other team to make them even.
/:cl:

### CODE REVIEW

File: code/modules/capture_the_flag/ctf_classes.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 	r_pocket = /obj/item/ammo_box/magazine/recharge/ctf/shotgun
 	belt = null
 	icon_state = "ctf_assaulter"
-	class_description = "Close combat class. Armed with a shotgun and rocket gloves."
+	class_description = "Close combat class. Armed with a shotgun and rocket gloves. Has significantly lower shield values due to higher moving speed."
```
Comment (Thunder12345) [inline code/modules/capture_the_flag/ctf_classes.dm:77]: 👍 This definitely needs highlighted

File: code/modules/capture_the_flag/ctf_equipment.dm [hunk 2]
```diff
@@ -207,7 +197,7 @@
 	///Icon state to be fed into the shielded component
 	var/team_shield_icon = "shield-old"
 	var/max_charges = 150
-	var/recharge_start_delay = 20 SECONDS
+	var/recharge_start_delay = 8 SECONDS
```
Comment (Thunder12345) [inline code/modules/capture_the_flag/ctf_equipment.dm:200]: 😕 Not spending longer waiting for your shield to regen than you would dying and respawning makes sense, but 8 seconds might be a bit short. 12 maybe?

File: code/modules/capture_the_flag/ctf_equipment.dm [hunk 3]
```diff
@@ -229,8 +219,7 @@
 	greyscale_config = /datum/greyscale_config/ctf_light
 	greyscale_config_worn = /datum/greyscale_config/ctf_light/worn
 	slowdown = -0.25
-
-	max_charges = 30
+	max_charges = 50
```
Comment (Thunder12345) [inline code/modules/capture_the_flag/ctf_equipment.dm:222]: 👎 Assaulters are meant to be a high-risk high-reward playstyle, only being able to take one hit before losing your shield is a fair price for being able to do the same to other classes by shotgunning them in the face

File: code/modules/clothing/outfits/ert.dm [hunk 4]
```diff
@@ -150,9 +150,9 @@
 	id_trim = /datum/id_trim/centcom/official
 	uniform = /obj/item/clothing/under/rank/centcom/official
 	back = /obj/item/storage/backpack/satchel
+	box = /obj/item/storage/box/survival
```
Comment (Thunder12345) [inline code/modules/clothing/outfits/ert.dm:153]: This isn't related to the CTF changes, you should put these fixes in their own separate PR so they aren't held up by the balance aspects of this PR.

File: code/modules/clothing/outfits/ert.dm [hunk 5]
```diff
@@ -181,9 +181,7 @@
 
 	back = /obj/item/mod/control/pre_equipped/responsory/inquisitory/commander
 	r_hand = /obj/item/nullrod/vibro/talking/chainsword
-	backpack_contents = list(
-		/obj/item/storage/box/survival = 1,
-	)
+	backpack_contents = null
```
Comment (Thunder12345) [inline code/modules/clothing/outfits/ert.dm:184]: Ditto here

**Review by Thunder12345 — CHANGES_REQUESTED** [review]

File: code/modules/capture_the_flag/ctf_equipment.dm (hunk 3 above)
Comment (Helg2) [reply code/modules/capture_the_flag/ctf_equipment.dm:222]: Are 2 hits instead of 1 to break shield makes such difference?

File: code/modules/capture_the_flag/ctf_equipment.dm (hunk 2 above)
Comment (Helg2) [reply code/modules/capture_the_flag/ctf_equipment.dm:200]: 10 maybe as a compromice? 12 isn't round number and just feels too close too respawning time

Comment (NamelessFairy) [reply code/modules/capture_the_flag/ctf_equipment.dm:200]: Remember with respawning time you also need to factor in walking to the front lines in addition to the delay for respawning.

**Comment by NamelessFairy** [comment]
I'm not a fan of how the assaulter class functions at all to be honest. Glass cannons just aren't that much fun to play against, they offer very little in the realm of interesting tactics. Increasing its health to make it less glass but equally as much of a cannon I heavily disagree with. If you want to raise their health then lower their damage so they're incapable of insta-killing fully armored players.

**Comment by Helg2** [comment]
I don't have anymore ideas, so it's useless to keep it draft

File: code/modules/capture_the_flag/ctf_controller.dm [hunk 6]
```diff
@@ -115,7 +115,7 @@
 		if(user.ckey in enemy_team_members)
 			to_chat(user, span_warning("No switching teams while the round is going!"))
 			return FALSE
-		if(friendly_team_members.len > enemy_team_members.len)
+		else if(friendly_team_members.len > enemy_team_members.len)
```
Comment (Ghommie) [inline code/modules/capture_the_flag/ctf_controller.dm:118]: I'm not sure this changes anything, since the above control statement already calls an early return, meaning that, if the condition `user.ckey in enemy_team_mebers` is true, this control statement won't run anyway.

Comment (Helg2) [reply code/modules/capture_the_flag/ctf_controller.dm:118]: image
I managed to get this message as a Guest, so maybe it just doesn't matter for anyone with ckey,  as i can't get it when logged in.

Comment (Ghommie) [reply code/modules/capture_the_flag/ctf_controller.dm:118]: It's perhaps something else that sends both messages, then, tho technically guests shouldn't be playing the server.
